controllers/clusteroperator.go

```diff
@@ -189,3 +193,25 @@ func (r *ProvisioningReconciler) updateCOStatus(newReason StatusReason, msg, pro
 
 	return r.syncStatus(co, conds)
 }
+
+func SetCOInDisabledState(osClient osclientset.Interface, version string) error {
+	//The CVO should have created the CO
+	co, err := osClient.ConfigV1().ClusterOperators().Get(context.Background(), clusterOperatorName, metav1.GetOptions{})
+
+	if err != nil {
+		return fmt.Errorf("failed to get clusterOperator %q: %v", clusterOperatorName, err)
+	}
+
+	conds := defaultStatusConditions()
+	v1helpers.SetStatusCondition(&conds, setStatusCondition(OperatorDisabled, osconfigv1.ConditionTrue, string(ReasonUnsupported), "Nothing to do on this Platform"))
```

Comment:
@wking I tried to incorporate some of your suggestions from https://github.com/openshift/cluster-baremetal-operator/pull/61.